samples/04_gis_analysts_data_scientists/chennai_floods_analysis.ipynb

```diff
@@ -790,7 +790,7 @@
    "source": [
```

Comment:
<p>Use the cell below to filter imagery based on the temporal conditions and export the filtered results as local images. These local images can then be compared with other time ranges. You can either use the where clause e.g.&nbsp;<code>where="(Year = " + str(start_year) + ")",</code>&nbsp;or use the temporal filter as shown below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>